### PR TITLE
Wrap processed stylesheets (close #626)

### LINKED ISSUE #626: Wrap processed stylesheets
Currently we processed scripts covered with processing comments:
```js
/*hammerhead|script-processing-header|start*/
...
/*hammerhead|script-processing-header|end*/
```

but for stylesheets we have only header:
```css
/* stylesheet processed via hammerhead */
...
```

Can we have same type of wrapper for stylesheets? Also can you expose prefix and postfix comment constants in client API for hammerhead? It's extremely urgent for https://github.com/DevExpress/testcafe/issues/574

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @inikulin

### CODE REVIEW

File: src/processing/script/header.js [hunk 1]
```diff
@@ -7,11 +7,13 @@ import reEscape from '../../utils/regexp-escape';
...
 import INSTRUCTION from './instruction';
 
-const PREFIX  = '/*hammerhead|script-processing-header|start*/';
-const POSTFIX = '/*hammerhead|script-processing-header|end*/';
 
-export const HEADER = [
-    PREFIX,
+export const SCRIPT_PROCESSING_START_COMMENT      = '/*hammerhead|script-processing-comment|start*/';
+export const SCRIPT_PROCESSING_END_COMMENT        = '/*hammerhead|script-processing-comment|end*/';
+export const SCRIPT_PROCESSING_END_HEADER_COMMENT = '/*hammerhead|script-processing-comment|end-header*/';
```
Comment (inikulin) [inline src/processing/script/header.js]: Maybe `/*hammerhead|script|start*/`, `/*hammerhead|script|processing-header-end*/`, `/*hammerhead|script|end*/`?

File: src/processing/style.js [hunk 2]
```diff
@@ -4,51 +4,55 @@
...
 
+import reEscape from '../utils/regexp-escape';
 import INTERNAL_ATTRS from '../processing/dom/internal-attributes';
 
-const SOURCE_MAP_REG_EX              = /#\s*sourceMappingURL\s*=\s*[^\s]+(\s|\*\/)/i;
-const CSS_URL_PROPERTY_VALUE_PATTERN = /(url\s*\(\s*)(?:(')([^\s']*)(')|(")([^\s"]*)(")|([^\s\)]*))(\s*\))|(@import\s+)(?:(')([^\s']*)(')|(")([^\s"]*)("))/g;
+const SOURCE_MAP_RE                       = /#\s*sourceMappingURL\s*=\s*[^\s]+(\s|\*\/)/i;
+const CSS_URL_PROPERTY_VALUE_PATTERN      = /(url\s*\(\s*)(?:(')([^\s']*)(')|(")([^\s"]*)(")|([^\s\)]*))(\s*\))|(@import\s+)(?:(')([^\s']*)(')|(")([^\s"]*)("))/g;
+const STYLESHEET_PROCESSING_START_COMMENT = '/*hammerhead|stylesheet-processing-comment|start*/';
+const STYLESHEET_PROCESSING_END_COMMENT   = '/*hammerhead|stylesheet-processing-comment|end*/';
```
Comment (inikulin) [inline src/processing/style.js]: Same for stylesheet

**Comment by inikulin** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Pull request merged**